packages/fleet/lib/db/migrations/20251101124200_remove_not_nullable_from_profiling_and_tracing.ts

```diff
@@ -0,0 +1,18 @@
+import type { Knex } from 'knex';
+import { NODE_CONFIG_OVERRIDES_TABLE } from '../../models/node_config_overrides.js';
+
+export async function up(knex: Knex): Promise<void> {
+    await knex.raw(`
+        ALTER TABLE ${NODE_CONFIG_OVERRIDES_TABLE} 
+        ALTER COLUMN is_tracing_enabled DROP NOT NULL,
+        ALTER COLUMN is_profiling_enabled DROP NOT NULL
+    `);
+}
```

Comment:
why dropping the not null in a separate query instead of setting it as nullable when adding the column